frontend/src/metabase/query_builder/components/notebook/Notebook.module.css

```diff
@@ -0,0 +1,8 @@
+.notebookRoot {
+  position: relative;
+  padding: 1rem;
+
+  @media screen and (--breakpoint-min-sm) {
```

Comment:
@nemanjaglumac Not sure but probably everything here can be done with styled props https://v6.mantine.dev/styles/style-props/#responsive-styles

Reply:
Shouldn't we have `breakpoints` defined in the `theme.ts` for that to work?

Reply:
https://github.com/metabase/metabase/pull/40903

https://github.com/metabase/metabase/pull/40694/commits/31f8a49e4d71a336607ff2e9b74cf85b725eed62